### DISCUSSION TITLE
restrequest4delphi com nuvemfiscal.com

### DISCUSSION THREAD

**Discussion by MrWolfX360** [viniciussanchez/RESTRequest4Delphi#119] [Q&A]
Alguém consegue usar o nuvemfiscal.com.br, com restrequest4delphi?

Eu tenho os json todos no postman, mas quando vou usar dá erro de autorização !!!

Tentei de várias formas !!!

Agradeço a atenção

**Comment by viniciussanchez** [comment]
Como está fazendo?
